### DISCUSSION TITLE
GUI Font Size and Colors

### DISCUSSION THREAD

**Discussion by hraP93SZrh** [monocasual/giada#492] [General]
Is there a way to alter the GUI to black text on a white background?

And/or increase the font sizes?

Thanks for any hints.

**Comment by gvnnz** [comment]
Hi @hraP93SZrh , the font/UI resize feature will be available in #178 , a mandatory step for our 1.0 milestone. There's no plan to change the theme instead (for now).

**Reply by hraP93SZrh** [reply]
Thank you, kindly.

**Reply by CTBlaze** [reply]
I'd still suggest to improve contrast, here and there, Like on the config dialog, the inactrive tabs are almost non-readable, esp. from a bit further away!, also greyed out options - hard to read! Another idea is volume knob on sample channel, also hard to see exact position due to low contrast of the colours. 
(Idea: could a number be added, at least as tooltip for mouseover?)
Here is a screenshot, but I realize it's kinda scaled here... on my laptop with 2.8k 14"  everything is hardly readable, esp. from a bit further away!, with WIndows 11 native scaling set to 200%
![image](https://user-images.githubusercontent.com/99221765/154371548-95d609df-2f5a-4329-a6d3-c4828a5d78da.png)

**Reply by gvnnz** [reply]
Hey @CTBlaze , I agree the overall contrast should be increased. Let's keep this in the roadmap, #178 is getting closer now that FLTK 1.4 (the library we use for UI) is about to be released.
